Update German translation

bomm opened the pull request:
I updated the German translation.

Lots of missing translations added for new messages, some inconsistencies fixed, some wrong translations fixed, some messages re-phrased.

bomm committed:
update German translation

bomm committed:
compiled after changing last translator

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
+#: easy_abc.py:2113
 msgid "This executable is used to display the music"
-msgstr "Dieses Programm wird verwendet um die Musik anzuzeigen"
+msgstr "Dieses Programm wird verwendet, um die Musik anzuzeigen"
```
bessere Übersetzung: "music" -> "Noten"

bomm replied:
changing to "Partitur"

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
+#: easy_abc.py:2909
 msgid "Right click the mouse to remove all choices"
-msgstr "Klicken Sie mit der rechten Maustaste um alle Markierungen zu entfernen"
+msgstr ""
```
Why an empty line here?

bomm replied:
probably a mistake

bomm replied:
In `poedit`, this is shown as a single line. It seems to be a continuation string, similar to `"foo" "bar"` in C. Of course it is strange to have this empty string, but I don't know why `poedit` generates this. 

While working on this, I checked the context. The English term "choices" refers to the available (previously used) values in the combobox. I will fix the translation and also fix the confirmation message which is currently wrong in English.

I think this message and the related confirmation message "This will remove the selections in the combobox" should be rephrased in English because it is too technical. (Why should a musician know what a combobox is?)

Maybe something like "Right click the mouse to remove all previously used format files from this list" and "This will remove all previously used file names from the list of choices."

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
 msgstr ""
-"Bögen am ersten Paar von Sechzehnteln verwenden (nützlich für einige 16tel Polkas)"
+"Bögen am ersten Paar von Sechzehnteln verwenden (nützlich für einige 16tel "
+"Polskas)"
```
Feststehender Begriff: "Polka" ohne 's'

bomm replied:
The English text mentions "polskas".  That's why I changed the translation. ~Unfortunately I don't know if the function described here is related to Polska or Polka.~

Polska is a Swedish dance in 3/4 measure which may have a "strange" rhythm sounding similar to 5/8. See https://en.wikipedia.org/wiki/Polska_(dance) or listen https://youtu.be/2llFjWyENfA?feature=shared&t=164

I just noticed that exactly this "Farmors brudpolska" shown in the YouTube video has sequences of 4 beamed 1/16 notes whereof the first two are connected with a slur. "Farmors brudpolska" means "grandmother's bridal/wedding polska" (more specifically Father's mother) = "Vatermutters Brautpolska"

![Farmors brudpolska](https://github.com/user-attachments/assets/eac9d691-a4c1-44aa-9d35-80def251b2a4)

bomm replied:
The window with the message about polskas is only shown when there is no path defined in the settings for `midi2abc` and you import a MIDI file. 

Unfortunately, when I import an exported MIDI file of Farmors brudpolska using these settings, the notes look very different from the original, and there is no group of 4 beamed 1/16 notes that might get the slur. :-(

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
-msgstr "&Speichern\tCtrl+S"
+#: easy_abc.py:6464
+msgid "&Save"
+msgstr "&Speichern (Strg+S)"
```
Is the shortcut really needed in the text???

bomm replied:
The keyboard shortcuts were present in the previous translation. They are not my invention, and I don't know if all of them are correct. First, I removed a few of them. Later I thought it might be good to show the keyboard shortcut as a help for the user. 
I'm open for discussion.

bomm replied:
I just checked the menu (on Ubuntu 24.04.2). It does not make sense to add the keyboard shortcuts to the text because they are shown anyway, and currently they are shown twice. I will remove the keyboard shortcuts.

hkiel replied:
That's what I wanted to say. Only I had no Linux or Windows machine to check if the short cuts are added there automatically as well.

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
-msgstr "Vollständige ABC-Spezifikation (englisch)"
+#: tune_actions.py:1322
+msgid "High Q"
+msgstr ""
```
Übersetzung ist leer!

bomm replied:
~I hope that~ an empty translation is replaced with the original text. ~I could not find where I can see this text in the application.~
"High Q" is the first value in a list of MIDI drum instruments in function `MidiDrumInstrumentChangeAction` at line 1320 of `tune_actions.py`. I don't want to translate the MIDI instrument names, because I think this might lead to confusion, and I don't know the correct translation for all of them.

see https://en.wikipedia.org/wiki/General_MIDI#GS_extensions

bomm replied:
I found it. When I create a drum notation, I get some instrument mapping lines like
~~~
%%MIDI drummap ^a 49
~~~
When I select such a `%%MIDI drummap` line, I get the list of instruments shown as the original message as expected, not an empty translation.

The German Wikipedia page lists only English names for the percussion instruments, although there are some differences between the  two Wikipedia pages and between these and EasyABC.

hkiel commented on locale/de/LC_MESSAGES/easyabc.po:
```diff
@@ -2,2798 +2,3492 @@ msgid ""
+
+#: tune_actions.py:1872
+msgid "Combine using beam"
+msgstr "Verbinde mit Balken"
```
Bindebogen???

bomm replied:
I did not check the context. Balken is beam, Bindebogen is slur, Haltebogen is tie. 
See https://www.myriad-online.com/resources/docs/harmony/english/tieslurbeam.htm

hkiel commented:
@bomm Good work! Can you please have a look at my review comments?

bomm commented:
> @bomm Can you please have a look at my review comments?

@hkiel  Where can I find the review comments? I'm used to review comments in Bitbucket, but I don't see anything here. Maybe I simply overlooked it.
I searched for [documentation](https://docs.github.com/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request), and found that you might have to submit the review to make the comments visible for everyone.

hkiel commented:
@bomm Sorry, I forgot to commit the review. You can see it now.

bomm committed:
Update German translation after review

bomm committed:
compiled German message file

bomm commented:
Work in progress...
I did not remove the keyboard shortcuts yet.

bomm committed:
remove superfluous keyboard shortcuts (probably from older English text)

bomm committed:
recompile German messages

bomm commented:
I removed the keyboard shortcuts. I think they were part of the original English messages, and the messages have been changed and assigned to the unchanged translations by fuzzy search.

I also removed some quotes when referring to header lines to make the messages consistent and added two comments regarding empty translations and shortened text.

Please check again.
